### PR TITLE
Create Select Action Page

### PR DESCRIPTION
**Pull request by Andrewsopoku**
**Issue**

- Link: https://github.com/NebraLtd/maker-starter-app/issues/55
- Summary: Display actionable items.

**Screenshots**

IMG_0544

- [ ] Tests added
- [ ] Cleaned up commit history (rebase!)
- [ ] Documentation added
- [ ] Thought about variable and method names

### CODE REVIEW

**Comment by marvinmarnold** [comment]
@Andrewsopoku can you grey out the options that are not yet fully implemented?

**Comment by marvinmarnold** [comment]
Other than my comment above LGTM. But I'll let @posterzh review.

**Comment by posterzh** [comment]
@marvinmarnold @Andrewsopoku Ok, I will review before integrating.

**Comment by marvinmarnold** [comment]
Actually I just launched the app and have some more feedback:

- The cancel button looks greyed out. @Louisreed any color suggestions to make it look more active?
- I also think clicking anywhere on the hotspot list entry should open the action items popup. Currently, you have to click on the three dots.

**Comment by Louisreed** [comment]
> 

@marvinmarnold We could use the primary colour for the button and make the text white. Also if you don't want to draw to much attention to it the opacity could be reduced by 50%. See attached

CreateSelectActionPage

**Comment by Andrewsopoku** [comment]
@marvinmarnold my recent push has Transfer Hotspot and Assert Location disabled.

I am looking into the other issues.

File: src/store/hotspots/hotspotsSlice.ts [hunk 1]
```diff
@@ -73,7 +73,6 @@ const hotspotsSlice = createSlice({
   extraReducers: (builder) => {
     builder.addCase(fetchHotspotsData.fulfilled, (state, action) => {
       state.hotspots = handleCacheFulfilled({ data: action.payload.hotspots })
-      console.log(`sggs ${state.hotspots}`)
```
Comment (posterzh) [inline src/store/hotspots/hotspotsSlice.ts:76]: Needs to clean up this log?

**Review by posterzh — APPROVED** [review]
You made a looking home page. I want to merge this PR so that I can integrate with minor conflict. IMO, some style fix can be done later.

**Comment by posterzh** [comment]
@Andrewsopoku While pulling the hotspots owned by a user, the home page shows that "You have 0 Hotspot(s)" message. Can you replace it to some user friendly message?
image

**Pull request merged**